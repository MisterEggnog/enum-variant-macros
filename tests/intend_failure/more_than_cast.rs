use enum_variant_macros::*;
#[derive(TryFromVariants)]
enum Variants {
	Shell(i32),
	Wite(f32),
	Ell(i32),
}
fn main() {
}
