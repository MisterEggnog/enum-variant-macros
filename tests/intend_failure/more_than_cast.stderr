error: Repeat of variant for type i32 in variant Variants::Ell. Already defined with Variants::blarg
 --> tests/intend_failure/more_than_cast.rs:6:6
  |
6 |     Ell(i32),
  |         ^^^
